samples/exceptions/ExceptionSamples.java

```diff
@@ -0,0 +1,35 @@
+package exceptions;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class ExceptionSamples {
+    public static void run() {
+
+        int a = 15;
+        int b = 0;
+
+        try {
+            System.out.println(divide(10, 0));
+        } catch (ArithmeticException e) {
+            System.out.println("Cannot divide by 0");
```

Comment:
**Very important thing** - When catching exception always remember to **not** skip exception details `e` in logs. So it should be logged like this:
`System.out.println("Cannot divide by 0. " + e);`